core/src/main/java/io/questdb/cairo/sql/Record.java

```diff
@@ -280,6 +280,14 @@ default short getShort(int col) {
         throw new UnsupportedOperationException();
     }
 
+    default @Nullable Utf8Sequence getSplitVarcharA(int col) {
```

Comment:
nit: we should add a javadoc here to clarify when split varchar should be preferred and what to expect from the returned value.